tox.ini

```diff
@@ -37,11 +40,11 @@ commands =
     coverage xml
 
 [testenv:isort]
-basepython = python3.4
+basepython = python3
```

Comment:
Locally I do not have a `python3`. This was the reason why I used `python3.4` here.